templates/controllers/tab/settings/navigationMenus/form/navigationMenuSettingsForm.tpl

```diff
@@ -0,0 +1,32 @@
+{**
+ * controllers/tab/settings/navigationMenus/form/NavigationMenuSettingsForm.tpl
```

Comment:
This isn't actually a form. It's just a tab. Might need to be renamed and moved.